libs/design/select/src/select/select.component.ts

```diff
@@ -327,23 +301,22 @@ export class DaffSelectComponent<T = unknown> extends DaffFormFieldControl<strin
 
     if (!this.openDirective.open) {
       this.openDirective.open = true;
-      this._animationState = getAnimationState(this.openDirective.open);
+
       if (this._value) {
         this._highlighted = this.options.findIndex((v) => v === this._value);
       }
-
-      // Get the width of the button element to use for the overlay
-      const buttonWidth = this.buttonElement.nativeElement.getBoundingClientRect().width;
+      const formFieldEl = <HTMLElement>this.elementRef.nativeElement.closest('daff-form-field');
```

Comment:
Can we access the `formField`'s `elementRef instead?

Reply:
I made updates. Form field's `elementRef` has to be public for this to work.